source/leuven/services/openondemand.rst

```diff
@@ -47,7 +52,8 @@ This also means that all your jobs should be submitted as **Slurm** jobs.
 For more detail on how to run jobs on wICE, check out our 
 :ref:`quick start guide<wice_t2_leuven>`.
 
-The jobs tab has two menus, 'Active Jobs' and 'Job Composer':
+The jobs tab has three menus, 'Active Jobs', 'Job Composer' and 'Projects' (which you cannot do much with it
```

Comment:
Perhaps mention that 'projects'  is planned feature for one of the next upstream OOD updates?

Reply:
done